frontend/src/components/Login/TownSelection.tsx

```diff
@@ -34,6 +41,7 @@ export default function TownSelection({ doLogin }: TownSelectionProps): JSX.Elem
   const [newTownIsPublic, setNewTownIsPublic] = useState<boolean>(true);
   const [townIDToJoin, setTownIDToJoin] = useState<string>('');
   const [currentPublicTowns, setCurrentPublicTowns] = useState<CoveyTownInfo[]>();
+  const [appearance, setAppearance] = useState<integer>(0);
```

Comment:
Discussed about making this a string instead and using `misa-[color]-hair` to easily communicate with backend types